demo/src/main/java/dev/materii/pullrefresh/demo/sample/DragRefreshSample.kt

```diff
@@ -31,13 +34,22 @@ fun DragRefreshSample(
             )
         }
     ) {
-        Box(
-            modifier = Modifier
-                .fillMaxSize()
-                .verticalScroll(rememberScrollState()),
-            contentAlignment = Alignment.Center
+        LazyColumn(
+            Modifier.fillMaxSize(),
```

Comment:
We use named arguments
```suggestion
            modifier = Modifier.fillMaxSize(),
```